index.js

```diff
@@ -67,6 +67,54 @@ function toggleColor(element) {
     }
 }
 
+const headerHText = document.querySelector('h1#header')
+
+headerHText.addEventListener('click', () => {
+    toggleColor(headerHText)
+})
+
 /***** Deliverable 2 *****/
+const profileLikeButton = document.querySelector("#profile > button.like-button")
+
+function addLike() {
+    let likeTextLine = document.querySelector("#profile > p.likes")
+    // let numberOfLikes = parseInt((likeTextLine.textContent).split(" ")[0])
+    // let likeWord = (likeTextLine.textContent).split(" ")[1]
+
+    // numberOfLikes ++
+    // newTextLine = `${numberOfLikes} ${likeWord}`
+
+    // likeTextLine.textContent = newTextLine
+    traveler.likes ++
+    likeTextLine.textContent = `${traveler.likes} Likes`
```

Comment:
goodgoodgood bc we're not updating the actual data, just what we're seeing on the DOM!